extension-localization/src/test/java/com/mapbox/maps/extension/localization/SupportedLanguagesTest.kt

```diff
@@ -18,8 +18,10 @@ private val commonLanguageList = listOf(
   arrayOf(NAME_JA, Locale.JAPANESE),
   arrayOf(NAME_KO, Locale.KOREA),
   arrayOf(NAME_ZH_HANS, Locale.SIMPLIFIED_CHINESE),
-  arrayOf(NAME_ZH_HANS,
```

Comment:
Should it be amended with previous commit?

Reply:
Maybe I miss some doc, but why it should be amended with prev commit?

Reply:
General rule is to have easy-to-read commit history but in general they're all squashed when PR is merged to `main`.